RavenDB.Identity/IdentityBuilderExtensions.cs

```diff
@@ -7,53 +7,37 @@
 
 namespace Raven.Identity
 {
-    /// <summary>
-    /// Extends the <see cref="IServiceCollection"/> so that RavenDB services can be registered through it.
-    /// </summary>
-    public static class IdentityBuilderExtensions
+	/// <summary>
+	/// Extends <see cref="IdentityBuilder"/> so that RavenDB services can be registered through it.
+	/// </summary>
+	public static class IdentityBuilderExtensions
     {
-        /// <summary>
-        /// Registers a RavenDB as the user store.
-        /// </summary>
-        /// <typeparam name="TUser">The type of user. This should be a class you created derived from <see cref="IdentityUser"/>.</typeparam>
-        /// <param name="services"></param>
-        /// <param name="setupAction">Identity options configuration.</param>
-        /// <returns>The identity builder.</returns>
-        public static IdentityBuilder AddRavenDbIdentity<TUser>(this IServiceCollection services, Action<IdentityOptions> setupAction = null)
```

Comment:
Is this a breaking change?

Reply:
Yes. It can't be done that way anymore due to the switch to `Microsoft.Extensions.Identity.Core`. 

EF does it this way also.

https://github.com/aspnet/AspNetCore/blob/fde783911b0b671dc4307ef5bcb2a9bc5e4a51bf/src/Identity/EntityFrameworkCore/src/IdentityEntityFrameworkBuilderExtensions.cs#L24

It also doesn't force the user to use `AddDefaultIdentity`. They can also configure their roles if they want.

```cs
services
	.AddRavenDbAsyncSession(CreateDocumentStore())
	.AddDefaultIdentity<User>()
	.AddRoles<Role>()
	.AddRavenDbStores<User, Role>();
```

Reply:
Thanks for the clarification. I'll merge this in and publish a new package.